docs/data_onboarding/linux.md

```diff
@@ -19,69 +19,70 @@ Installation and Configuration Guide:
 Insert the input stanzas below for the Splunk Add-on for Linux and Unix (*nix) to collect auditing data. Each stanza should be placed in the inputs.conf file (create a local directory if necessary): 
 
 
-    [script://./bin/df.sh] 
-    interval = 3600 
-    sourcetype = df 
-    source = df 
-    disabled = 0 
-    index = os 
-
-    # Lists users who could login (i.e., they are assigned a login shell) 
-    [script://./bin/usersWithLoginPrivs.sh] 
-    sourcetype = usersWithLoginPrivs 
-    source = usersWithLoginPrivs 
-    interval = 3600 
-    disabled = 0 
+    [script://./bin/df.sh]
+    interval = 3600
+    sourcetype = df
+    source = df
+    disabled = 0
+    index = os
+
+    # Lists users who could login (i.e., they are assigned a login shell)
+    [script://./bin/usersWithLoginPrivs.sh]
+    sourcetype = usersWithLoginPrivs
+    source = usersWithLoginPrivs
+    interval = 3600
+    disabled = 0
     index = os
 
     # Shows stats per link-level Etherner interface (simply, NIC) 
     [script://./bin/interfaces.sh] 
```

Comment:
I think most of these inputs requires root user permission

Reply:
We have that note for TA-cyences section on the same page, so fine for now.